### ISSUE TITLE
Serve all 3rd party assets from local versions

### ISSUE THREAD

**Issue by AnalogJ** [fastenhealth/fasten-onprem#159]
> I'd be nice if 3rd party assets were embedded. You've got multiple things loading from Google, Fontawesome, and Fastenhealth. For a self-hosted system, I'm not expecting any 3rd party assets.
> https://old.reddit.com/r/DataHoarder/comments/12pg3bu/fasten_health_open_source_selfhosted_personal/jl0m2il/

- [ ] FontAwesome
- [ ] Google Fonts

**Comment by ericlathrop** [comment]
Yeah, the site looks broken unless I disable my ad blocker (uBlock origin, set to block 3rd party assets). I shouldn't have to do that for something self-hosted.
